CATALOG.md

```diff
@@ -7,27 +7,27 @@ Depending on the CNF type, not all tests are required to pass to satisfy best pr
 
 ## Test cases summary
 
-### Total test cases: 88
+### Total test cases: 90
```

Comment:
Is this change creating a new test case? (from what I can observe, it's not the case)
I'm saying this because there are currently 89 test cases and here it is said that there are 90, but I cannot see what's the new test... probably because it's not taking into account the removal of affiliated-certification-container-is-certified from one of the PRs merged on these last days?

Reply:
Yeah this PR needs a rebase + regenerate catalog to remove the CATALOG.md change from this PR.